libr/db/table.c

```diff
@@ -56,6 +71,10 @@ const char *r_db_table_field_i(struct r_db_table_t *table, int elem) {
 }
 
 void *r_db_table_free(struct r_db_table_t *table) {
-	free(table);
+	if (table->name) free (table->name);
```

Comment:
see manpage, it is stnadard and all platforms i have tried, even QNX, old Solaris or Mac versions do. and this is because all static analysis tools complain about this